Protos.podspec

```diff
@@ -0,0 +1,135 @@
+#
+#  Be sure to run `pod spec lint Protos.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see https://guides.cocoapods.org/syntax/podspec.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |spec|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  These will help people to find your library, and whilst it
+  #  can feel like a chore to fill in it's definitely to your advantage. The
+  #  summary should be tweet-length, and the description more in depth.
+  #
+
+  spec.name         = "Protos"
+  spec.version      = "1.0.0"
+  spec.summary      = "The protocol buffer specification to create the Cyface binary format"
+
+  # This description is used to generate tags and improve search results.
+  #   * Think: What does it do? Why did you write it? What is the focus?
+  #   * Try to keep it short, snappy and to the point.
+  #   * Write the description between the DESC delimiters below.
+  #   * Finally, don't worry about the indent, CocoaPods strips it!
+  spec.description  = <<-DESC
+This Project contains the measurement.proto file.
+It serves as input description for the Cyface binary format, starting with version 2.
```

Comment:
Shouldn't it be binary version 3 now?

Reply:
That is why I wrote "starting with". I did release a version for the old faulty V2, for testing.